### PR TITLE
ComfyUI + Flux AI training(ai-toolkit)

### PR DESCRIPTION
**Pull request by johnnynunez**
Credits: https://github.com/ostris/ai-toolkit, https://github.com/comfyanonymous/ComfyUI
@dusty-nv I've updated Pytorch and their libraries, and opencv and bitsandbytes required for FLUX.
Now pytorch has heterogeneus memory with +12.2 so you can use swap

### CODE REVIEW

**Comment by johnnynunez** [comment]
finally @dusty-nv  ai-toolkit is working
image

File: packages/opencv/config.py [hunk 1]
```diff
@@ -35,17 +35,17 @@ def opencv(version, requires=None, default=False, url=None):
     
 package = [
     # JetPack 6
-    opencv('4.8.1', '==36.*', default=True),
+    opencv('4.10.0', '==36.*', default=True),
```
Comment (dusty-nv) [inline packages/opencv/config.py:38]: If comfyui/flux requires opencv 4.10, can you keep 4.8 as the default, and then make add `opencv:4.10.0` to the depends list of whatever container it is that needs opencv 4.10

File: packages/cuda/cudnn/config.py [hunk 2]
```diff
@@ -9,7 +9,7 @@
 else:
     if L4T_VERSION.major >= 36:
         if CUDA_VERSION >= Version('12.4'):
-            CUDNN_VERSION = Version('9.0')
+            CUDNN_VERSION = Version('9.3')
```
Comment (dusty-nv) [inline packages/cuda/cudnn/config.py:12]: @johnnynunez can cuDNN 9.0 be kept as default?

**Comment by dusty-nv** [comment]
Thank you good sir @johnnynunez!  even added CUDA 12.6 🤣

I added some comments to your PR about changing the default versions of cuDNN and OpenCV.  After resolving those, should be g2g to merge 👍

**Pull request merged**